action.yaml

```diff
@@ -46,26 +52,52 @@ inputs:
     description: 'Secret name to pull env variables from AWS Secret Manager'
     required: false
     default: ''
+  
+  # Application
+  app_directory:
+    description: 'Relative path for the directory of the app (i.e. where `Dockerfile` and `docker-compose.yaml` files are located). This is the directory that is copied to the EC2 instance.  Default is the root of the repo.'
   app_port:
     description: 'Port to expose for the app'
     required: false
-    default: '3000'
   lb_port:
     description: 'Load balancer listening port. Defaults to 80 if NO FQDN provided, 443 if FQDN provided'
     required: false
   lb_healthcheck: 
     description: 'Load balancer health check string. Defaults to HTTP:app_port'
     required: false
+  
+  # EC2 Instance
   ec2_instance_profile:
     description: 'The AWS IAM instance profile to use for the EC2 instance'
   ec2_instance_type: 
     description: 'The AWS Instance type'
     required: false
-    default: 't2.small'
```

Comment:
why was this removed?

Reply:
All defaults have been moved as close to terraform as possible. In this case, the default is handled in the variables.tf file

Reply:
Defaults that could be have been moved as close to the terraform variables.tf file as possible. In this case, the default has been moved exactly there.

Reply:
right, but why?